package.json

```
@@ -2,19 +2,24 @@
   "name": "root",
   "version": "10.0.0",
   "private": true,
-  "engines": {
-    "node": ">=14.0.0"
-  },
-  "scripts": {
-    "lint:eslint": "yarn eslint . --ext ts,js",
-    "lint:config-validation": "node ./scripts/validate-configs.js",
-    "lint:misc": "prettier '**/*.json' '!**/rules-snapshot.json' '**/*.md' '!**/CHANGELOG.md' '**/*.yml' --ignore-path .gitignore",
-    "lint": "yarn lint:eslint && yarn lint:misc --check && yarn lint:config-validation",
-    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn lint:config-validation --write"
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/eslint-config.git"
   },
   "workspaces": [
     "packages/*"
   ],
+  "scripts": {
+    "lint": "yarn lint:eslint && yarn lint:misc --check && yarn lint:config-validation",
+    "lint:changelogs": "yarn workspaces foreach --parallel --verbose run lint:changelog",
+    "lint:config-validation": "node ./scripts/validate-configs.js",
+    "lint:eslint": "yarn eslint . --ext ts,js",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn lint:config-validation --write",
+    "lint:misc": "prettier '**/*.json' '!**/rules-snapshot.json' '**/*.md' '!**/CHANGELOG.md' '**/*.yml' --ignore-path .gitignore"
```

One more addition from the module template:

```suggestion
    "lint:misc": "prettier '**/*.json' '!**/rules-snapshot.json' '**/*.md' '!**/CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore"
```